shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/networking/KubernetesNetworkingService.java

```diff
@@ -79,7 +80,7 @@ private Ingress buildIngress(BridgeIngress bridgeIngress, Service service) {
 
         HTTPIngressPath httpIngressPath = new HTTPIngressPathBuilder()
                 .withBackend(ingressBackend)
-                .withPath("/" + service.getMetadata().getName() + PATH_REGEX)
+                .withPath("/" + service.getMetadata().getName() + EVENTS_ENDPOINT_SUFFIX + PATH_REGEX)
```

Comment:
This works nice locally with Kubernetes. I have tested creating bridges and see the endpoint shown for `GET` bridges is `http://i.p.address/events` and if I create, for example a Slack Action, I can `POST` to `http://i.p.address/events` perfectly well. Woot.